android/src/main/java/io/ably/lib/types/RegistrationToken.java

```diff
@@ -33,4 +33,12 @@ public String toName() {
             return name().toLowerCase();
         }
     }
+
+    @Override
```

Comment:
We've been using `[ ]` instead of `{ }`: https://github.com/ably/ably-java/blob/main/lib/src/main/java/io/ably/lib/types/Message.java#L103

Reply:
Looking at this and from hazy memory, I'm pretty sure I used IntelliJ IDEA's boilerplate via auto-complete to create this method... I'm happy to change and will push a commit to do so. Is this a formal Ably standard, or just convention within this codebase?

Reply:
We have very few `toString()` implementations across this codebase, however those already differ in terms of style. e.g.:

- perhaps more illustrative of the style you point out: [ErrorInfo](https://github.com/ably/ably-java/blob/0e4da0f8ef63524b6fdc72e3d1a80561acf329d1/lib/src/main/java/io/ably/lib/types/ErrorInfo.java#L68)
- more recently illustrative of the style I used: [MessageExtras](https://github.com/ably/ably-java/blob/0e4da0f8ef63524b6fdc72e3d1a80561acf329d1/lib/src/main/java/io/ably/lib/types/MessageExtras.java#L126)

My preference would be to align with IntelliJ IDEA's default, thus leave this PR in current state and address square bracket "contraventions" later. Any reason that's a bad choice, @paddybyers ?

Reply:
> Any reason that's a bad choice

No, so long as we're consistent I'm fine